src/Conf.ml

```diff
@@ -1574,16 +1574,6 @@ let parse_line config ~from s =
       , from )
```

Comment:
You could remove the `from` parameter. Also, splitting the string is not usefull because the pattern only match lists of one element, you could use strip instead.